pkgs/development/ocaml-modules/ocaml-lsp/jsonrpc.nix

```
@@ -10,10 +10,14 @@
 }:
 
 let params =
-  if lib.versionAtLeast ocaml.version "4.13"
+  if lib.versionAtLeast ocaml.version "4.14"
   then {
-    version = "1.10.3";
-    sha256 = "sha256-o6wQc7Byi5T0vbARF3LAq69/9wMkOZRQ6rcVa/rBUfE=";
+    version = "1.11.1";
+    sha256 = "sha256-jqiqGB6wXSXtWyqOAS/zZNPa0ezHthQdeLtYI0gc1qQ=";
+  } else if lib.versionAtLeast ocaml.version "4.13"
+  then {
+    version = "1.10.5";
+    sha256 = "sha256-TeJS6t1ruWhWPvWNatrnSUWI6T17XKiosHLYizBDDcw=";
   } else if lib.versionAtLeast ocaml.version "4.12"
   then {
     version = "1.9.0";
```

```suggestion
    version = "1.9.1";
```
https://github.com/ocaml/ocaml-lsp/tree/1.9.1

Fixes
* Disable functionality reliant on ocamlformat-rpc for now (https://github.com/ocaml/ocaml-lsp/pull/555)
* 4.13 compatiblity